blog-posts/optimze-gatsby-urls.md

```diff
@@ -0,0 +1,153 @@
+---
+path: "/blog/how-to-optimize-gatsby-urls/"
+date: "2021-07-15"
+title: "How consistency helps you to optimize Gatsby URLs for users and search engines"
```

Comment:
path and title are different. We might want to create the path/slug automatically from our title in the future? Because this creates, wait for it... inconsistency in urls. Some match the title, some are not 😅

Reply:
Oh no, not again 😄 I changed the url, it's now like the title